lib/app/constants/theme_colors.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+
+class ThemeColors {
```

Comment:
:thought_balloon: I think we should use flutters built-in theme logic for this. Since your PR should be merged as soon as possible to avoid blocking further development, I think this should be done in a separate issue.
See https://docs.flutter.dev/cookbook/design/themes

Same for the text styles below.

Reply:
Good idea, I will change this.

Reply:
I changed the code so we use the flutter built-in theme logic.